test/test_bids_examples.py

```diff
@@ -5,7 +5,8 @@
 import bids2openminds.converter
 
 example_dataset = [("ds003", 13), ("ds000247", 6),
```

Comment:
can you make a comment and explain what the numbers are in the example dataset tuples?